src/app/components/PersonalisedContent/index.tsx

```diff
@@ -0,0 +1,98 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+
+import { Article } from '#app/models/types/optimo';
+import CurationGrid from '#app/components/Curation/CurationGrid';
+import Subheading from '#app/components/Curation/Subhead';
+import { Summary } from '#app/models/types/curationData';
+import { EventTrackingData } from '#app/lib/analyticsUtils/types';
+import useViewTracker from '#app/hooks/useViewTracker';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import styles from '#app/components/RelatedContentSection/index.styles';
+
+const PersonalisedContent = ({
+  pageData,
+  personalisedTopicCurationExperimentVariant,
+}: {
+  pageData: Article;
+  personalisedTopicCurationExperimentVariant: string;
+}) => {
+  type PersonalisedContentType = {
+    title?: string;
+    summaries?: Summary[];
+    curationLength?: number;
+    id?: string;
+    link?: string;
+    renderVisuallyHiddenH2Title?: boolean;
+    curationSubheading?: string;
+    topicId?: string;
+  };
+
+  const personalisedContentArray = pageData.secondaryColumn
+    ?.personalisedContent as PersonalisedContentType[] | undefined;
+
+  const getPersonalisedContentData = () => {
+    if (personalisedTopicCurationExperimentVariant !== 'personalised') {
```

Comment:
checking the variant here seems redundant as we also check it in the ArticlePage file 
`personalisedContentExperimentVariant === 'personalised',` where 'showPersonalisedContent' is false if the variant is not 'peronalised'. (I know this will be changed later to the actual variant names, but still it seems the check is done twice)